app/routes/events/components/RegistrationMeta.tsx

```diff
@@ -58,7 +58,7 @@ const ConsentStatus = ({
   if (!useConsent) return null;
 
   if (photoConsents && allConsentsAnswered(photoConsents)) {
-    const { WEBSITE, SOCIAL_MEDIA } = PHOTO_CONSENT_DOMAINS;
+    const { WEBSITE, SOCIAL_MEDIA } = PhotoConsentDomain;
```

Comment:
Is it not bad practice to destruct an enum?

Reply:
Nah, it works just like an object, just with proper types. So it doesn't really matter

Reply:
I just felt like it kinda defeated its purpose, even thought it kinda doesn't ..